README.md

```diff
@@ -17,55 +17,86 @@ The path to dynawo deploy is then the path to the subdirectory `dynawo` in the d
 PATH_TO_DYNAWO_DEPLOY=${DYNAWO_HOME}/deploy/gcc8/shared/dynawo/
 ```
 
-Then generate the cmake cache in your build directory:
+To build DynaFlow Launcher you need to clone the repository and launch the following commands in the source code directory, it will create a `myEnvDFL.sh` file that will be your personal entrypoint to launch DFL and parametrise some options.
```

Comment:
typo: parametrise

Reply:
OK. This typo also exists in dynawo readme